config/locales/en.yml

```diff
@@ -1772,8 +1772,8 @@ en:
         acknowledgement: "I have read and acknowledged all information under the 'Register' tab, including entry fees, refund policies and any extra requirements listed above."
         next_step: "Continue to next Step"
       update:
-        being_updated: "Registration is being updated"
-        being_deleted: "Registration is being deleted"
+        being_updated: "Registration is being updated..."
+        being_deleted: "Registration is being deleted..."
```

Comment:
How are we handling `...` ellipsis in other locale string?
I know there is a single Unicode `…` character as opposed to three `.` characters

Reply:
seems to be all `...` and no `…`